src/app/features/test-records/views/amended-test-record/amended-test-record.component.html

```diff
@@ -3,14 +3,13 @@
     <div class="govuk-notification-banner__header">
       <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">Important</h2>
     </div>
+
     <div class="govuk-notification-banner__content">
       <p class="govuk-notification-banner__heading">Amended Test Record - created at {{ testResult.createdAt | date: 'dd/MM/yyyy HH:mm' }}</p>
     </div>
   </div>
 
   <app-base-test-record [edit]="(edit$ | async) || false" [testResult]="testResult"></app-base-test-record>
 
-  <section class="section">
-    <app-defects [edit]="(edit$ | async) || false" [defectsData$]="defectsData$"></app-defects>
-  </section>
+  <app-defects *ngIf="defects$ | async as defects" class="section" [isEditing]="(edit$ | async) || false" [defects]="defects"></app-defects>
```

Comment:
Would it be possible to add app-defects inside of `base-test-record` component since it is used by both the amended and "normal" test record component?

Reply:
good idea